deploy/chart/templates/daemonset.yaml

```diff
@@ -31,14 +35,14 @@ spec:
       tolerations:
       {{- toYaml . | nindent 8 }}
       {{- end }}
-      hostNetwork: true
+      hostNetwork: {{ default true .Values.enableHostNetwork }}
       containers:
         - image: {{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           name: ccm-linode
           args:
             - --leader-elect-resource-lock=leases
-            - --v=3
+            - --v={{ default 2 .Values.logVerbosity }}
```

Comment:
Any specific reason we want to change the previous default of `3` to `2`?

Reply:
It was by accident. One of the internal manifests uses verbosity of 2. I reverted the change.